.github/workflows/macdebug.yml

```diff
@@ -28,21 +28,60 @@ jobs:
       CXX: clang++
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
 
-   # Enable tmate debugging of manually-triggered workflows if the input option was provided
+   
+    - name: Install Dependencies on MacOS
+      run: |    
+        brew update
+        brew install libtiff open-mpi libyaml ccache
+        echo "CMAKE_PREFIX_PATH=/usr/local/opt/libomp" >> $GITHUB_ENV
+
+    - name: Install Tensorflow API on MacOS
+      uses: UCL/install-tensorflow-action@main
+      with:
+        version: 2.11.0
+        os: darwin
+
+    - name: Select Python 3.10
+      # otherwise turtlebrowser/get-conan@v1.1 fails on macos-12
+      # ref: https://github.com/turtlebrowser/get-conan/issues/4
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+
+    - name: Install Conan
+      id: conan
+      uses: turtlebrowser/get-conan@main
+      with:
+        version: 1.59.0
+    
+    - name: Checkout cppflow repo
+      uses: actions/checkout@v3
+      with:
+        repository: UCL/cppflow.git
+        path: cppflow
+        ref: master
+
+    - name: Add Tensorflow paths to env
```

Comment:
This step was me trying to fix the issue that is now solved in `cppflow` so should probably remove this.